core/trino-main/src/main/java/io/trino/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -113,8 +113,10 @@
  */
 class AggregationAnalyzer
 {
+    private final List<Expression> groupByExpressions;
     // fields and expressions in the group by clause
-    private final Set<FieldId> groupingFields;
+    @Nullable
+    private Set<FieldId> groupingFields;
```

Comment:
The commit message specified `groupByExpression` is lazily evaluated should we update the message ?

Reply:
my bad, will fix it briefly